packages/angular/pwa/pwa/index_spec.ts

```diff
@@ -169,4 +169,52 @@ describe('PWA Schematic', () => {
       expect(swFlag).toBeTrue();
     });
   });
+
+  describe('@angular-devkit/build-angular:application builder', () => {
+    function convertBuilderToLegacyBrowser(): void {
```

Comment:
Nit: This name seems wrong? I assume "LegacyBrowser" would refer to `@angular-devkit/build-angular:browser`. Should it be "LegacyApplication" or "ApplicationFromDevkit"?

Alternatively you could just inline this into `beforeEach` and avoid the naming problem altogether.